<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Taiga Config Wizard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
	font: bold 12px;
qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0))
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HaveFinishButtonOnEarlyPages|QWizard::HaveNextButtonOnLastPage|QWizard::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="page1_locateDir">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Locate Taiga Directory</string>
    </property>
    <widget class="QLabel" name="lblLocateDirHelp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Papyrus</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;We will need to know where to place the conf files. &lt;br /&gt;Please select bin folder of your Taiga instal. &lt;br /&gt;This folder will contain the Taiga executables. &lt;br /&gt; Such as OpenSim.exe, OpenSim.Server.exe.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>401</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditTaigaDir">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnBrowseDirectory">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page2_serverUrls">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Server urls</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="rightMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayoutSimNetwork">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>User server URL:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditSimUserUrl">
          <property name="accessibleName">
           <string>user_server_url</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Messaging server URL:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditSimMessagingUrl">
          <property name="accessibleName">
           <string>messaging_server_url</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Grid server URL:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditSimGridUrl">
          <property name="accessibleName">
           <string>grid_server_url</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Asset server URL:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditSimAssetUrl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="accessibleName">
           <string>inventory_server_url</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Inventory server URL:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditSimInventoryUrl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="accessibleName">
           <string>asset_server_url</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditSimPort">
          <property name="accessibleName">
           <string>sim_http_port</string>
          </property>
          <property name="inputMask">
           <string>000000; </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>HTTP Port</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRobust">
          <property name="text">
           <string>&amp;Robust conf.</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page3_addRegions">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Add Regions</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>471</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left:10px;</string>
        </property>
        <property name="text">
         <string>Current Regions</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listWidgetRegions">
          <property name="accessibleName">
           <string>sim_regions</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAddRegion">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdateRegion">
            <property name="text">
             <string>&amp;Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveRegion">
            <property name="text">
             <string>&amp;Remove Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page5_database">
   <widget class="QGroupBox" name="groupBoxDB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>211</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Database Access</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="minimumSize">
             <size>
              <width>59</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGridDBUser">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>root</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGridDBPassword">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>root</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>105</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelShowErrors">
          <property name="text">
           <string>Data Source</string>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDataSource">
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Select database plugin:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboDBPlugin">
          <property name="accessibleName">
           <string>sim_database_plugin</string>
          </property>
          <item>
           <property name="text">
            <string>OpenSim.Data.MySQL.dll</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OpenSim.Data.SQLite.dll</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Using database names: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetDBNames"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnChangeDataBaseNames">
        <property name="text">
         <string>&amp;Change database names..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>512</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Physics</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Physics plugin:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPhysicsPlugin">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>sim_physics_plugin</string>
        </property>
        <item>
         <property name="text">
          <string>RexOpenDynamicsEngine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BasicPhysicsPlugin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>basicphysics</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="labelSimDBString">
        <property name="font">
         <font>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="accessibleName">
         <string>sim_db_connection_string</string>
        </property>
        <property name="text">
         <string>simDBStr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGridDBString">
        <property name="font">
         <font>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="accessibleName">
         <string>grid_db_connection_string</string>
        </property>
        <property name="text">
         <string>gridDBStr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSimDBPlugin">
        <property name="accessibleName">
         <string>sim_database_plugin</string>
        </property>
        <property name="text">
         <string>simDBPlg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGridDbPlugin">
        <property name="accessibleName">
         <string>grid_database_plugin</string>
        </property>
        <property name="text">
         <string>gridDbPlg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGridNhib">
        <property name="accessibleName">
         <string>grid_nhibernate</string>
        </property>
        <property name="text">
         <string>gridNHib</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPropProv">
        <property name="accessibleName">
         <string>property_provider</string>
        </property>
        <property name="text">
         <string>propProv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelModRexConnStr">
        <property name="accessibleName">
         <string>modrex_db_connection_string</string>
        </property>
        <property name="text">
         <string>modRexDB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="page6_complete">
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Complete</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>381</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Save current:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSaveName"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveConfigs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Save config..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>About to write following files:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetFilesToWrite"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditTaigaDir</tabstop>
  <tabstop>btnBrowseDirectory</tabstop>
  <tabstop>lineEditSimPort</tabstop>
  <tabstop>lineEditSimUserUrl</tabstop>
  <tabstop>lineEditSimMessagingUrl</tabstop>
  <tabstop>lineEditSimGridUrl</tabstop>
  <tabstop>lineEditSimAssetUrl</tabstop>
  <tabstop>lineEditSimInventoryUrl</tabstop>
  <tabstop>checkBoxRobust</tabstop>
  <tabstop>listWidgetRegions</tabstop>
  <tabstop>btnAddRegion</tabstop>
  <tabstop>btnUpdateRegion</tabstop>
  <tabstop>btnRemoveRegion</tabstop>
  <tabstop>lineEditGridDBUser</tabstop>
  <tabstop>lineEditGridDBPassword</tabstop>
  <tabstop>lineEditDataSource</tabstop>
  <tabstop>btnChangeDataBaseNames</tabstop>
  <tabstop>listWidgetFilesToWrite</tabstop>
  <tabstop>btnSaveConfigs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
